src/Sylius/Bundle/ProductBundle/Resources/config/services.xml

```diff
@@ -22,6 +22,10 @@
         <parameter key="sylius.generator.product_variant.class">Sylius\Bundle\ProductBundle\Generator\VariantGenerator</parameter>
 
         <parameter key="sylius.validator.product.unique.class">Sylius\Bundle\ProductBundle\Validator\ProductUniqueValidator</parameter>
+
+        <parameter key="sylius.model.product_association.class">Sylius\Component\Product\Model\ProductAssociation</parameter>
```

Comment:
Defining these parameters should be handled by association bundle, why we need them here?